### DISCUSSION TITLE
Issue with Velero Backup on EKS Fargate

### DISCUSSION THREAD

**Discussion by mxAlexKing** [vmware-tanzu/velero#7989] [Community support Q&A]
I am experiencing an issue with configuring Velero backups on my EKS Fargate environment. Despite setting the Helm chart to exclude the installation of the node-agent and to avoid including DaemonSets in the backup list, I am still encountering partially failed backups.

I am only interested in backing up Kubernetes resources, not the contents of Persistent Volumes.

**Error Details:**
```
Phase:  PartiallyFailed (run `velero backup logs dev-backup-manual-08072024-01` for more information)

Errors:
  Velero:    name: /jenkins-test-1-8bkt5 error: /daemonset pod not found in running state in node fargate-ip1.ec2.internal
             name: /jenkins-test-2-87655 error: /daemonset pod not found in running state in node fargate-ip2.ec2.internal
             name: /jenkins-test-3-b88tk error: /daemonset pod not found in running state in node fargate-ip3.ec2.internal

  Cluster:    <none>
  Namespaces: <none>
```

I have verified my Helm chart configurations to ensure that the node-agent is not installed and that DaemonSets are excluded. However, the issue persists.

Could you please assist me in resolving this problem? Any insights or additional configuration steps I might have missed would be greatly appreciated.

K8s versions:
```
Client Version: v1.29.1
Server Version: v1.29.4-eks-036c24b
```

Velero versions:
```
Client:
        Version: v1.11.1
        Git commit: bdbe7eb242b0f64d5b04a7fea86d1edbb3a3587c
Server:
        Version: v1.11.0
```

Thank you for your assistance.

**Comment by blackpiglet** [comment]
Please make sure there is not any PodVolumeBackup involved in the backup.
